### ISSUE TITLE
Implement new web style props parity with RN core in Fabric

### ISSUE THREAD

**Issue by jonthysell** [microsoft/react-native-windows#11184]
### Summary

RN is trying to make it easier to share code/knowledge between React web dev and React Native dev. Part of this means implementing web style props in React Native.

Upstream tracking issue: https://github.com/facebook/react-native/issues/34425

### Motivation

We need to maintain priority with RN core's props, though not necessarily for Paper as we lean into Fabric.

### Basic Example

RN Core implemented borderBlock colors here: https://github.com/facebook/react-native/commit/597a1ff60b3e1844b4794fb4acd40fa073f2e93b

**Comment by chrisglein** [comment]
Some of the JS ones we'll get for free.
On Fabric some of them might come for free with the border resolve helper thing.
But we'll need to run through the checklist in the ☂️ issue, see which we need to implement and what we get for free. And navigate which are for Paper and which are Fabric only. It's more of a goal to get these working for Fabric than for Paper.

**Comment by chrisglein** [comment]
@danielayala94 , Jon had opened this issue awhile ago but I think your detailed inventory will replace this, agree?

**Comment by danielayala94** [comment]
> @danielayala94 , Jon had opened this issue awhile ago but I think your detailed inventory will replace this, agree?

Yes. Many of the props alluded in the RN Core issue are already being tracked in the spreadsheet.
